src/providers/MeetingProvider/docs/MeetingManager.stories.mdx

```diff
@@ -51,21 +28,38 @@ import { MeetingManagerConfig } from 'amazon-chime-sdk-component-library-react';
 (data: MeetingJoinData) => Promise<void>
 
 interface MeetingJoinData {
-  // The response of calling chime:CreateMeeting from your server.
-  meetingInfo: any;
-
-  // The response of calling chime:CreateAttendee from your server.
-  attendeeInfo: any;
-
-  // The kind of device for which the browser requests permission.
-  // Or override the default device label trigger in the Amazon Chime SDK for JavaScript.
-  deviceLabels?: DeviceLabels | DeviceLabelTrigger;
-
-  // Override the default event reporter in the Amazon Chime SDK for JavaScript.
-  eventController?: EventController;
-
-  // Set the meeting config. This will override the props from MeetingProvider.
-  meetingManagerConfig?: MeetingManagerConfig;
+  // Use the meeting session configuration.
+  // The default properties of the meeting session configuration can be updated to customize the meeting as per the builder's preference.
+  // Reference Meeting Session Configuration properties in Amazon Chime SDK for JavaScript: https://aws.github.io/amazon-chime-sdk-js/classes/meetingsessionconfiguration.html
+  meetingSessionConfiguration: MeetingSessionConfiguration;
+
+  options ?: {
```

Comment:
nit: Remove an extra space between options and `?`.

Reply:
Done